### PR TITLE
Clean up FAQ and some related pages

### PR DESCRIPTION
**Pull request by david-a-wheeler**
This focuses on cleaning up/updating the Metamath FAQ, but also makes some tweaks to mm.html and other.html (e.g., to move some material around).

This adds a clear obituary for Norm in the FAQ.
He deserves the recognition, and it'll make the situation clearer to newcomers.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -999,86 +802,115 @@
...
+<B>A:</B>
+Norman "Norm" Dwight Megill, Ph.D. (1950-2021)
+was the original creator of Metamath.
+He created the Metamath language, developed the first tool to
+support it (now called metamath-exe), and first developed axioms
+and proofs in this language.
+For over 30 years he cultivated an international community of
+people with the shared dream of digitizing and verifying mathematics.
+His ideas and design have been influential in formal mathematics.
+He had interest in properties of Quantum logic and Hilbert spaces,
```
Comment (david-a-wheeler) [inline index.html]: Add "an" interest.

File: index.html [hunk 2]
```diff
@@ -641,18 +429,29 @@
...
-<LI><I>metamath-exe</I> - the 
+<li><b><i>metamath-exe</i></b> - the 
   <A HREF="mmprog">metamath-exe</A> program is an
   ASCII-based ANSI C program with a command-line interface.
   This is the original program for processing Metamath; today it's
   it's often called "metamath-exe" to distinguish it from other Metamath tools.
-<LI><I>yamma</I> -
+<li><b><i>yamma</i></b> -
   Yamma is a Metamath proof assistant for Visual Studio Code.
   It's implemented as a vSCode extension for Metamath.
```
Comment (expln) [inline index.html]: `vSCode`

I think it should be `VS Code`
This page https://code.visualstudio.com/ uses `VS Code` in many places.

Comment (david-a-wheeler) [reply index.html]: Thanks!! I'll make that fix.

**Comment by david-a-wheeler** [comment]
I fixed the problems reported. Anyone else?

File: index.html [hunk 3]
```diff
@@ -999,86 +802,115 @@
...
-<LI>If you want to modify a web page,
-send email to <A HREF="http://us.metamath.org/email.html">Norm Megill</A>.
+go to its
+<a href="https://github.com/digama0/mmj2">mmj2 GitHub repository</a>.
+<LI>If you want to modify the metamath-exe program (the original tool
+implementation written in C), go to the
+<A href="https://github.com/metamath/metamath-exe">metamath-exe
+GitHub repository</A>.
+
+<LI>If you want to modify a webpage, you'll to determine its sourse.
```
Comment (jkingdon) [inline index.html]: Appears to be missing the word "need" betweeen "you'll" and "to determine".

**Review by jkingdon — APPROVED** [review]

**Comment by david-a-wheeler** [comment]
Thanks everyone!

**Pull request merged**